easy-ui-tokens/src/color/status.json

```diff
@@ -0,0 +1,11 @@
+{
+  "color": {
+    "status": {
+      "system": { "value": "{color.blue.800.value}" },
+      "info": { "value": "{color.blue.500.value}" },
+      "success": { "value": "{color.green.500.value}" },
+      "warning": { "value": "{color.purple.500.value}" },
+      "error": { "value": "{color.red.500.value}" }
+    }
+  }
+}
```

Comment:
do you recall what the plan was for the status colors? i know we documented them in figma, but curious if the previous design team communicated what their intention was with these. 🤔

Reply:
i think they're supposed to be meant for warning messages, error alerts, etc; but yeah, i don't know if there was anything specific communicated about them by the design team

Reply:
i think its good to keep in 👍